desc/objectives/linear_objectives.py

```diff
@@ -89,6 +89,8 @@ def build(self, eq, use_jit=True, verbose=1):
             idx = np.arange(eq.surface.R_basis.num_modes)
         else:  # specified modes
             modes = np.atleast_2d(self._modes)
+            if np.isfortran(modes):
```

Comment:
what is this needed for? I think ``isfortran`` is deprecated (https://numpy.org/doc/stable/reference/generated/numpy.isfortran.html)

Could we just always make it a contiguous array if its really needed?